src/HelpersServiceProvider.php

```diff
@@ -64,4 +70,82 @@ protected function extendValidator()
                 ->exists();
         });
     }
+
+    protected function extendRouter()
+    {
+        if (!method_exists(Route::class, 'whereIn')) {
```

Comment:
```suggestion
        // Backward compatibility with Laravel < 8
        if (!method_exists(Route::class, 'whereIn')) {
```

Reply:
Done, only I wrote version 9 instead of 8.